jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -4,172 +4,199 @@
 
 /** Status React component used to display Juju status. */
 class Status extends React.Component {
+  /**
```

Comment:
That's right, they don't need to access state, but they are necessary for the component and if they're a component level method they won't clash with other modules if they're scoped to the component.